src/patternfly/components/OptionsMenu/examples/options-menu-align-right-example.hbs

```diff
@@ -0,0 +1,6 @@
+{{#> options-menu options-menu--IsExpanded="true" id="options-menu-multiple-example" options-menu--label-text="Choose sort methods"  options-menu--modifier="pf-m-align-right"}}
```

Comment:
Similar to previous comment, we don't need the word "Choose" as part of this label. Also, this raises an interesting question of how do differentiate the sub-menus and associate labels with the sub-menus. I can try to find examples of this that we could follow, but I'm wondering if those labels would be "Sort direction" and "Sort by"?

Reply:
The Style/Color in this [wai-aria menu example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-2/menubar-2.html) is similar to what we have. They use nested lists, where the primary list is labeled by the toggle, and the nested lists have labels that describe their group of options (e.g. "Sort direction" and "Sort by" in our case).